ui/views/view.cc

```diff
@@ -1840,6 +1840,12 @@ void View::PropagateNativeThemeChanged(const ui::NativeTheme* theme) {
   OnNativeThemeChanged(theme);
 }
 
+void View::PropagateSoftVisibilityChanged(bool visible) {
+  for (int i = 0, count = child_count(); i < count; ++i)
```

Comment:
Don't know. As on 16 other places in that file (where I copy pasted the code) it is like this, suggest to keep the style.